src/App.tsx

```diff
@@ -50,7 +36,7 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector />
```

Comment:
The 'UserSelector' component is used correctly. Ensure that this component is properly defined and exported in './components/UserSelector'.